tests/everflow/everflow_test_utilities.py

```diff
@@ -426,7 +636,15 @@ def policer_mirror_session(self, duthosts, rand_one_dut_hostname, config_method)
         Yields:
             dict: Information about the mirror session configuration.
         """
-        duthost = duthosts[rand_one_dut_hostname]
+        duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+        vendor = duthost.facts["asic_type"]
+        hostvars = duthost.host.options['variable_manager']._hostvars[duthost.hostname]
+        # relocated because of sonic-buildimage issue #11826, leaving skip logic in testcase causes
+        # this fixture to run causing crash.
+        for asic in self.MIRROR_POLICER_UNSUPPORTED_ASIC_LIST:
+            vendorAsic = "{0}_{1}_hwskus".format(vendor, asic)
+            if vendorAsic in hostvars.keys() and duthost.facts['hwsku'] in hostvars[vendorAsic]:
+                pytest.skip("Skipping test since mirror policing is not supported on {0} {1} platforms".format(vendor, asic))
```

Comment:
Why not put this in `test_everflow_dscp_with_policer`?

Reply:
With the skip in the test case, the fixtures were still running and activating the policer setup.  This was hitting an open issue (sonic-buildimage 11826).